Week1/homework/app.js

```diff
@@ -1,11 +1,115 @@
 'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+
+  const booksArray = [
+    `The_Goldfinch`,
+    `A_Garden_of_Earthly_Delights`,
+    `Beloved`,
+    `There_Once_Lived_a_Woman_Who_Tried_to_Kill_Her_Neighbors_Baby`,
+    `The_House_of_the_Spirits`,
+    `The_Left_Hand_of_Darkness`,
+    `The_Unwomanly_Face_of_War`,
+    `Chernobyl_Prayer`,
+    `Runaway`,
+    `The_Year_of_Magical_Thinking`
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
-}
+  const booksObject = {
+    The_Goldfinch: {
+      title: `The Goldfinch`,
+      author: `Donna Tartt`,
+      original_language: `English`
+    },
+    A_Garden_of_Earthly_Delights: {
+      title: `A Garden of Earthly Delights`,
+      author: `Carol Oates`,
+      original_language: `English`
+    },
+    Beloved: {
+      title: `Beloved`,
+      author: `Toni Morrison`,
+      original_language: `English`
+    },
+    There_Once_Lived_a_Woman_Who_Tried_to_Kill_Her_Neighbors_Baby: {
+      title: `There Once Lived a Woman Who Tried to Kill Her Neighbo\'s Baby`,
+      author: `Lyudmila Petrushevskaya`,
+      original_language: `Russian`
+    },
+    The_House_of_the_Spirits: {
+      title: `The House of the Spirits`,
+      author: `Isabel Allende`,
+      original_language: `Spanish`
+    },
+    The_Left_Hand_of_Darkness: {
+      title: `The Left Hand of Darkness`,
+      author: `Ursula K. Le Guin`,
+      original_language: `English`
+    },
+    The_Unwomanly_Face_of_War: {
+      title: `The Unwomanly Face of War`,
+      author: `Svetlana Alexievich`,
+      original_language: `Russian`
+    },
+    Chernobyl_Prayer: {
+      title: `Chernobyl Prayer`,
+      author: `Svetlana Alexievich`,
+      original_language: `Russian`
+    },
+    Runaway: {
+      title: `Runaway`,
+      author: `Alice Munro`,
+      original_language: `English`
+    },
+    The_Year_of_Magical_Thinking: {
+      title: `The Year of Magical Thinking`,
+      author: `Joan Didion`,
+      original_language: `English`
+    }
+  };
+
+  for (const book of booksArray) {
```

Comment:
Don't forget to remove debugging code like this loop before submitting your homework.

Reply:
ok